stdlib/_dbm.pyi

```diff
@@ -22,20 +24,16 @@ if sys.platform != "win32":
         def __setitem__(self, key: _KeyType, value: _ValueType) -> None: ...
         def __delitem__(self, key: _KeyType) -> None: ...
         def __len__(self) -> int: ...
-        def __del__(self) -> None: ...
         def __enter__(self) -> Self: ...
         def __exit__(
             self, exc_type: type[BaseException] | None, exc_val: BaseException | None, exc_tb: TracebackType | None
         ) -> None: ...
         @overload
-        def get(self, k: _KeyType) -> bytes | None: ...
+        def get(self, k: _KeyType, /) -> bytes | None: ...
         @overload
-        def get(self, k: _KeyType, default: _T) -> bytes | _T: ...
+        def get(self, k: _KeyType, default: _T, /) -> bytes | _T: ...
         def keys(self) -> list[bytes]: ...
-        def setdefault(self, k: _KeyType, default: _ValueType = ...) -> bytes: ...
-        # Don't exist at runtime
-        __new__: None  # type: ignore[assignment]
-        __init__: None  # type: ignore[assignment]
```

Comment:
I think I'd vote for leaving these as they are, unless they're causing a concrete problem somewhere. If we do end up figuring out a way to run stubtest on this class, we can just allowlist them.

I don't know if mypy and pyright _actually_ infer that you can't create arbitrary instances of this class as a result of these assignments... but logically they _should_. I think it's the best we can do.

Reply:
Makes sense to me. As a situation, it feels very similar to types that can't be subclassed. It's too bad we don't have a decorator like @final to handle this type of thing.